build_tools/circle/doc_environment.yml

```diff
@@ -32,6 +32,7 @@ dependencies:
   - pooch
   - sphinxext-opengraph
   - sphinx-remove-toctrees
+  - sphinx-design
```

Comment:
@Charlie-XIAO could you please have a note somewhere to make sure all dependencies are documented and they're in `min_dependencies` if relevant, before merging to `main`?

Reply:
Do you mean in the code, or maybe I can add a todo-list in #28084?

Reply:
Add the TODO is what I meant.

Reply:
Done, it is in the top comment.